### PR TITLE
Hotfix for executable webscripts

### PR DESCRIPTION
**Pull request by LuKePicci**

### CODE REVIEW

File: 3main [hunk 1]
```diff
@@ -255,6 +255,7 @@ do
   if [[ ! -e "$HOME/www/cgi-bin/$webscript" ]]
   then
     ln -s "${NVOC}/www/cgi-bin/$webscript" "${HOME}/www/cgi-bin/$webscript"
+    chmod +x "${HOME}/www/cgi-bin/$webscript"
```
Comment (papampi) [inline 3main:258]: I already make it executable in git repo, but its safe to have this too

**Review by papampi — APPROVED** [review]

**Review by papampi — APPROVED** [review]

**Comment by papampi** [comment]
Miners sub module is too big, some thing is not right ...
First it downloaded 400Mb, then goes for getting more sources:

Will post git pull log when miner sub module fetch is done

**Comment by papampi** [comment]
@LuKePicci check this:
First downloaded 400Mb, then again downloaded over 500 Mb downloading each source again 

```
payam@payam-office:~$ git clone --single-branch --branch 19-2.1 --recursive https://github.com/papampi/nvOC_by_fullzero_Community_Release.git /home/payam/nvoc/testing
Cloning into '/home/payam/nvoc/testing'...
remote: Counting objects: 1821, done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 1821 (delta 27), reused 22 (delta 19), pack-reused 1786
Receiving objects: 100% (1821/1821), 1.04 MiB | 247.00 KiB/s, done.
Resolving deltas: 100% (1121/1121), done.
Checking connectivity... done.
Submodule 'miners' (https://github.com/papampi/nvOC_miners.git) registered for path 'miners'
Cloning into 'miners'...
remote: Counting objects: 649, done.
remote: Compressing objects: 100% (50/50), done.
remote: Total 649 (delta 17), reused 17 (delta 2), pack-reused 597
Receiving objects: 100% (649/649), 416.25 MiB | 483.00 KiB/s, done.
Resolving deltas: 100% (259/259), done.
Checking connectivity... done.
Submodule path 'miners': checked out '43562a120b76c789ad482f0eca76fae1a29eabf1'
Submodule 'ANXccminer/src' (https://github.com/anorganix/ccminer-phi.git) registered for path 'ANXccminer/src'
Submodule 'ASccminer/src' (https://github.com/alexis78/ccminer) registered for path 'ASccminer/src'
Submodule 'KTccminer-cryptonight/src' (https://github.com/KlausT/ccminer-cryptonight) registered for path 'KTccminer-cryptonight/src'
Submodule 'KTccminer/src' (https://github.com/KlausT/ccminer.git) registered for path 'KTccminer/src'
Submodule 'KXccminer/src' (https://github.com/krnlx/ccminer-xevan) registered for path 'KXccminer/src'
Submodule 'MSFTccminer/src' (https://github.com/MSFTserver/ccminer) registered for path 'MSFTccminer/src'
Submodule 'NAccminer/src' (https://github.com/Nanashi-Meiyo-Meijin/ccminer) registered for path 'NAccminer/src'
Submodule 'SPccminer/src' (https://github.com/sp-hash/ccminer) registered for path 'SPccminer/src'
Submodule 'TPccminer/src' (https://github.com/tpruvot/ccminer) registered for path 'TPccminer/src'
Submodule 'vertminer/src' (https://github.com/vertcoin-project/vertminer-nvidia) registered for path 'vertminer/src'
Cloning into 'ANXccminer/src'...
remote: Counting objects: 354, done.
remote: Total 354 (delta 0), reused 0 (delta 0), pack-reused 354
Receiving objects: 100% (354/354), 29.80 MiB | 369.00 KiB/s, done.
Resolving deltas: 100% (56/56), done.
Checking connectivity... done.
Submodule path 'miners/ANXccminer/src': checked out 'cd6fab68823e247bb84dd1fa0448d5f75ec4917d'
Cloning into 'ASccminer/src'...
remote: Counting objects: 4313, done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 4313 (delta 0), reused 1 (delta 0), pack-reused 4310
Receiving objects: 100% (4313/4313), 63.25 MiB | 795.00 KiB/s, done.
Resolving deltas: 100% (2839/2839), done.
Checking connectivity... done.
Submodule path 'miners/ASccminer/src': checked out 'aafe2d19b6d9eb07d942d70ced3049e9aed1c241'
Cloning into 'KTccminer-cryptonight/src'...
remote: Counting objects: 1982, done.
remote: Total 1982 (delta 0), reused 0 (delta 0), pack-reused 1982
Receiving objects: 100% (1982/1982), 22.28 MiB | 531.00 KiB/s, done.
Resolving deltas: 100% (1134/1134), done.
Checking connectivity... done.
Submodule path 'miners/KTccminer-cryptonight/src': checked out '31865e0b846ad98e2ce99aeb9b28bb51d3be22db'
Cloning into 'KTccminer/src'...
remote: Counting objects: 11841, done.
remote: Total 11841 (delta 0), reused 0 (delta 0), pack-reused 11841
Receiving objects: 100% (11841/11841), 118.91 MiB | 328.00 KiB/s, done.
Resolving deltas: 100% (8650/8650), done.
Checking connectivity... done.
Submodule path 'miners/KTccminer/src': checked out 'c5ab73837c8024f1e6b8fe7ad46e6881fb8366e6'
Cloning into 'KXccminer/src'...
remote: Counting objects: 325, done.
remote: Total 325 (delta 0), reused 0 (delta 0), pack-reused 325
Receiving objects: 100% (325/325), 1.84 MiB | 194.00 KiB/s, done.
Resolving deltas: 100% (44/44), done.
Checking connectivity... done.
Submodule path 'miners/KXccminer/src': checked out '7d41d49b92db27b9ab80270adaa92f6b06d1ef78'
Cloning into 'MSFTccminer/src'...
remote: Counting objects: 8070, done.
remote: Total 8070 (delta 0), reused 0 (delta 0), pack-reused 8070
Receiving objects: 100% (8070/8070), 64.89 MiB | 388.00 KiB/s, done.
Resolving deltas: 100% (5870/5870), done.
Checking connectivity... done.
Submodule path 'miners/MSFTccminer/src': checked out '78dad7dd659eae72a07d2448de62b1946c1f2b41'
Cloning into 'NAccminer/src'...
remote: Counting objects: 7994, done.
remote: Total 7994 (delta 0), reused 0 (delta 0), pack-reused 7993
Receiving objects: 100% (7994/7994), 73.17 MiB | 104.00 KiB/s, done.
Resolving deltas: 100% (5791/5791), done.
Checking connectivity... done.
Submodule path 'miners/NAccminer/src': checked out '8affcb9cd09edd917d33c1ed450f23400f571bdb'
Cloning into 'SPccminer/src'...
remote: Counting objects: 8970, done.
remote: Total 8970 (delta 0), reused 0 (delta 0), pack-reused 8970
Receiving objects: 100% (8970/8970), 102.31 MiB | 490.00 KiB/s, done.
Resolving deltas: 100% (6465/6465), done.
Checking connectivity... done.
Submodule path 'miners/SPccminer/src': checked out '9e86bdd24ed7911b698f1d0ef61a4028fcbd13c5'
Cloning into 'TPccminer/src'...
remote: Counting objects: 8412, done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 8412 (delta 0), reused 1 (delta 0), pack-reused 8407
Receiving objects: 100% (8412/8412), 65.06 MiB | 116.00 KiB/s, done.
Resolving deltas: 100% (6110/6110), done.
Checking connectivity... done.
Submodule path 'miners/TPccminer/src': checked out 'b54be470e14860a1da303d3eaf34598e1e16d9af'
Cloning into 'vertminer/src'...
remote: Counting objects: 559, done.
remote: Total 559 (delta 0), reused 0 (delta 0), pack-reused 559
Receiving objects: 100% (559/559), 39.50 MiB | 76.00 KiB/s, done.
Resolving deltas: 100% (224/224), done.
Checking connectivity... done.
Submodule path 'miners/vertminer/src': checked out '48b170a5828256600ca71e66d4c114af4e114236'

```

**Comment by LuKePicci** [comment]
Ok, it is needed to set the recursion depth to 1 in the clone command line to avoid fetching miner sources.

**Comment by papampi** [comment]
Possible to fetch the source only for re compile ?

**Comment by LuKePicci** [comment]
Sure, that was what I intended to achieve. I’m now updating the wiki with the new comands.

________________________________
Da: Payam Nab <notifications@github.com>
Inviato: Tuesday, May 29, 2018 12:25:02 PM
A: papampi/nvOC_by_fullzero_Community_Release
Cc: Luca Piccirillo; Mention
Oggetto: Re: [papampi/nvOC_by_fullzero_Community_Release] Hotfix for miners submodule and webscripts (#77)


Possible to fetch the source only for re compile ?

**Pull request merged**